tests/test_image.py

```diff
@@ -79,3 +79,22 @@ def test_stackstderr():
     m, n = stackstderr(x)
     assert_array_almost_equal(m, np.std(x, axis=0) / np.sqrt(n), 3)
     assert_array_equal(n, np.ones((20, 20), dtype=np.float32) * 1000.0)
+
+
+def test_images_mean():
+    x = []
+    for ii in range(1,11):
+        x.append(np.repeat(ii*np.ones(ii*100, dtype=np.float32), 400).reshape(
+        (ii*100, 20, 20)))
+    x = np.array(x)
+    m = images_mean(x)
+
+
+def test_images_sum():
+    x = []
+    for ii in range(1,11):
+        x.append(np.repeat(ii*np.ones(ii*100, dtype=np.float32), 400).reshape(
```

Comment:
This is very contrived ... Is there not a cleaner way to do this using ``np.ones``